mtp_api/apps/service/tests/test_downtime.py

```diff
@@ -4,66 +4,64 @@
 from django.test import TestCase
 from django.utils import timezone
 
-from service.models import Downtime, SERVICES
+from service.constants import Services
+from service.models import Downtime
 
 
 class DowntimeTestCase(TestCase):
-    def _downtime_is_active(self, service, start, end):
-        downtime = Downtime(service=service, start=start, end=end)
```

Comment:
Uh, does it mean this was wrong/broken?

Reply:
no, it wasn't broken. `<enum instance>.value` is basically the same as `<enum instance>` where a string is expected

Reply:
Oh because `TextChoices` is also a string, gotcha! I prefer the `.value` one as it's explicit/more clear